lib/active_model_serializers/serializable_resource.rb

```diff
@@ -2,7 +2,7 @@
 
 module ActiveModelSerializers
   class SerializableResource
-    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links, :serialization_context, :key_transform])
+    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links, :serialization_context, :key_transform, :jsonapi_include_toplevel_object])
```

Comment:
A question I have is what should be different between wanting to pass in your own `jsonapi` object vs. telling the adapter to include the default one?